google-cloud-core/src/main/java/com/google/cloud/Policy.java

```diff
@@ -158,65 +175,143 @@ protected Builder(Policy policy) {
      *
      * @throws NullPointerException if the given map is null or contains any null keys or values
      * @throws IllegalArgumentException if any identities in the given map are null
+     * @throws IllegalArgumentException if policy version is equal to 3 or has conditional bindings.
```

Comment:
This is uncommon though not unreasonable. Usually we give two reason in one @throws clause. I'm not sure I've ever seen this done before, and I don't know whether the javadoc tool supports it.